### ISSUE TITLE
Error when opening a file using the file browser pane

### ISSUE THREAD

**Issue by mdcutone** [psychopy/psychopy#3519]
Coder's file browser pane raise an error when opening a non-text file. Error is harmless but annoying.

**Comment by mdcutone** [comment]
The raised error looks like this:

```
Traceback (most recent call last):
  File "C:\Program Files\PsychoPy3\lib\site-packages\psychopy\app\coder\fileBrowser.py", line 571, in OnItemActivated
    self.coder.fileOpen(None, self.selectedItem.abspath)
  File "C:\Program Files\PsychoPy3\lib\site-packages\psychopy\app\coder\coder.py", line 2369, in fileOpen
    self.setFileModified(False)
  File "C:\Program Files\PsychoPy3\lib\site-packages\psychopy\app\coder\coder.py", line 2765, in setFileModified
    self.currentDoc.UNSAVED = isModified
AttributeError: 'NoneType' object has no attribute 'UNSAVED'

```

It only happens when there is no editor open.

**Comment by peircej** [comment]
Fixed in fe5e7a4
